reduce text in readme (#525)
simplex-chat/simplex-chat: README.md

- 🚀 [TestFlight preview for iOS](https://testflight.apple.com/join/DWuT2LQu) with the new features 1-2 weeks earlier - **limited to 10,000 users**!
- 🖥 Available as a terminal (console) app / CLI on Linux, MacOS, Windows.

## Why privacy of communications matter

> Everyone should care about privacy and security of their communications - ordinary conversations can put you in danger even if you are innocent.

Reviewer: I think it would be better to say something like "ordinary innocuous conversations can put you in danger" or "even if you have nothing to hide"

"even if you are innocent" somehow to me feels like you're pitching to the guilty and the innocents are a secondary user type

> One of the most shocking stories is the experience of [Mohamedou Ould Slahi](https://en.wikipedia.org/wiki/Mohamedou_Ould_Slahi) that he wrote about in his memoir and that is shown in The Mauritanian movie. He was put into Guantanamo camp, without trial, and was tortured there for 15 years after a phone call to his relative in Afghanistan, under suspicion of being involved in 9/11 attacks, even though he lived in Germany since 10 years prior to the attacks.

Reviewer: suggestion: "he lived in Germany for the 10 years leading up to the attacks."

> This is not enough to use an end-to-end encrypted messenger, we all should use the messengers that protect the privacy of our personal networks - who we are connected with.

Reviewer: ```suggestion
It is not enough to use an end-to-end encrypted messenger, we all should use messengers that protect the privacy of our personal networks - who we are connected with.
```

## SimpleX unique approach to privacy and security

### Full privacy of your identity, profile, contacts and metadata

**Unlike any other existing messaging platform, SimpleX has no identifiers assigned to the users** - not even even random numbers. This protects the privacy of who are you communicating with, hiding it from SimpleX platform servers and from any observers. [Read more](./docs/SIMPLEX.md#full-privacy-of-your-identity-profile-contacts-and-metadata).

### The best protection against spam and abuse

> As you have no identifier on SimpleX platform, you cannot be contacted unless you share a one-time invitation link or an optional temporary user address. [Read more](./docs/SIMPLEX.md#the-best-protection-against-spam-and-abuse).

Reviewer: Is it worth mentioning that you can change this so your contact details cannot be forwarded on without your consent.

### Complete ownership, control and security of your data

SimpleX stores all user data on client devices, the messages are only held temporarily on SimpleX relay servers until they are received. [Read more](./docs/SIMPLEX.md#complete-ownership-control-and-security-of-your-data).

### Users own SimpleX network

You can use SimpleX with your own servers and still communicate with people using the servers that are pre-configured in the apps or any other SimpleX servers. [Read more](./docs/SIMPLEX.md#users-own-simplex-network).

## For developers

> We plan that SimpleX platform will grow into the platform for any distributed Internet applications platform, allowing to build any services that people can access via chat, with custom web-based UI widgets that anybody with a basic HTML/CSS/JavaScript knowledge can create in a few hours.

Reviewer 2: could the second "platform" be by mistake?
```suggestion
We plan that SimpleX platform will grow into the platform for any distributed Internet applications, allowing to build any services that people can access via chat, with custom web-based UI widgets that anybody with a basic HTML/CSS/JavaScript knowledge can create in a few hours.
```

> We plan that SimpleX platform will grow into the platform for any distributed Internet applications platform, allowing to build any services that people can access via chat, with custom web-based UI widgets that anybody with a basic HTML/CSS/JavaScript knowledge can create in a few hours.

Reviewer: ```suggestion
We plan that the SimpleX platform will grow into the platform supporting any distributed Internet application. This will allow you to build any service that people can access via chat, with custom web-based UI widgets that anybody with basic HTML/CSS/JavaScript knowledge can create in a few hours.
```

You already can:

- use SimpleX Chat library to integrate chat functionality into your apps.

> - use SimpleX Chat bot templates in Haskell to build your own chat bot services (TypeScript SDK is coming soon)

Reviewer 2: ```suggestion
- use SimpleX Chat bot templates in Haskell to build your own chat bot services (TypeScript SDK is coming soon).
```
